### DISCUSSION TITLE
🎨 Data Symphony: Artistic Visualization with plotnine

### DISCUSSION THREAD

**Discussion by PablaOO7** [has2k1/plotnine#853] [2024 Plotnine Contest]
### Authors

Jaspreet Singh Pabla

### Full description

**README link:** https://github.com/PablaOO7/Plotnine-Contest-2024/blob/main/Second_Entry_Plotnine/README.md

### Plots: All plots are available in the repo in the png format.

1. [Abstract Art plot](https://github.com/PablaOO7/Plotnine-Contest-2024/blob/main/Second_Entry_Plotnine/abstract_art_plotnine.png)
2. [Vitruvian Data Visualization](https://github.com/PablaOO7/Plotnine-Contest-2024/blob/main/Second_Entry_Plotnine/enhanced_vitruvian_plot.png)
3. [Floral Pattern Plot](https://github.com/PablaOO7/Plotnine-Contest-2024/blob/main/Second_Entry_Plotnine/floral_pattern.png)
4. [Geometric Abstract plot](https://github.com/PablaOO7/Plotnine-Contest-2024/blob/main/Second_Entry_Plotnine/geometric_abstract.png)

### Code repository

**Code Link:** https://github.com/PablaOO7/Plotnine-Contest-2024/blob/main/Second_Entry_Plotnine/p9_AbstractArt_notebook.ipynb

**Comment by has2k1** [comment]
## Entry 1

![abstract_art_plotnine](https://github.com/user-attachments/assets/211d11b5-c464-449d-8a96-625367b2d009)

**Comment by has2k1** [comment]
## Entry 2

![enhanced_vitruvian_plot](https://github.com/user-attachments/assets/d9c66e46-9a11-42b3-9034-476354d8ec04)

**Comment by has2k1** [comment]
## Entry 3
![floral_pattern](https://github.com/user-attachments/assets/2712af5a-4976-4d18-9ced-d73c5334669d)

**Comment by has2k1** [comment]
## Entry 4
![geometric_abstract](https://github.com/user-attachments/assets/8c3b6893-e2e0-4943-a6b9-1a243116199b)

**Comment by has2k1** [comment]
@PablaOO7, The winning submission has been announced [here](https://posit.co/blog/winner-of-the-2024-plotnine-plotting-contest/). Thank you again for taking part in the contest.
